setup.py

```diff
@@ -46,7 +46,7 @@
 
 install_requires = [
     "Flask>=0.12",
-    "werkzeug>=0.16.1,<0.17",
+    "werkzeug>=2.0",
```

Comment:
Any particular reason why you change this? 
By the way, this is not effective unless you update `requirements.txt` as well.